apps/designer/app/designer/features/style-panel/shared/use-style-data.ts

```diff
@@ -58,13 +59,10 @@ export const useStyleData = ({
     () => cssRule?.style as Style
   );
 
-  const currentStyle = useMemo<Style>(
-    () => ({
-      ...selectedInstanceData?.browserStyle,
-      ...breakpointStyle,
-    }),
-    [selectedInstanceData?.browserStyle, breakpointStyle]
-  );
+  const currentStyle = useStyleInfo({
+    localStyle: breakpointStyle,
```

Comment:
can't we rename it everywhere consistently?

Reply:
Can be done later. Don't want to put all change here.